src/components/SocialShareButtons.vue

```diff
@@ -17,7 +17,7 @@
         class="social-button pinterest"
         target="_blank"
         @click="onSocialMediaLinkClick('Pinterest')"
-        :href="`https://www.pinterest.com/pin/create/bookmarklet/?media=${imageURL}&description=${shareText}`">
+        :href="`https://www.pinterest.com/pin/create/bookmarklet/?media=${image_URL}&description=${shareText}`">
```

Comment:
This change would cause tests to fail because `image_URL` is undefined. There is no definition in `SocialShareButtons` for a variable with this name.

I suspect that this is also the reason for your change to the unit tests for this component: The `media` parameter gets its value from the `imageURL` prop, which is mocked on like 17 of the test spec file. By breaking the connection between the prop, and the value in the URL, the test was also broken.

In case you revert this change, you should also revert your changes to the unit test spec file